catalog/spatial/registry/registry-api/src/main/java/org/codice/ddf/registry/api/RegistryStore.java

```diff
@@ -32,4 +32,10 @@
      * @return true if allowed otherwise false
      */
     boolean isPullAllowed();
+
+    /**
+     * Indicates the Id associated with this registry
+     * @return registry id in a string
+     */
+    String getRegistryId();
```

Comment:
Only as of last sprint (I believe) would it have been used and there was a caveat issued that development off that release will not be supported moving forward. It also has the experimental header so it shouldn't be too surprising if older implementations need to be updated.

Basically I'm hoping we aren't rolling over any toes.